### PR TITLE
Document how to connect to the SFTP server

### PR DESCRIPTION
**Pull request by capnrefsmmat**
Thanks to @WichadaLMK and @chinandrew for screenshots.

Preview: http://rosmarus.refsmmat.com/delphi-epidata/symptom-survey/server-access.html

### CODE REVIEW

**Comment by chinandrew** [comment]
Now the whole world can see that I misspelled my own name for the user on my computer.

File: docs/symptom-survey/server-access.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+---
+
+# SFTP Server Access
+
+Researchers with data use agreements to access the raw data from the COVID-19
+symptom survey can access the data over SFTP. (If you do not have a data use
+agreement, see the [main survey page](index.md) for information about getting
+access.)
+
+If you're not familiar with SFTP, it is a protocol for accessing and downloading
```
Comment (krivard) [inline docs/symptom-survey/server-access.md:14]:
```suggestion
If you're not familiar with SFTP, it is a protocol for securely accessing and downloading
```

File: docs/symptom-survey/server-access.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+   ![Screenshot of WinSCP's credentials and login screen.](screenshots/winscp-small2.png)
+
+4. You should then have access to the CSV files of individual responses from CMU
+   Delphi COVID Survey of Facebook users. See our [response files documentation](survey-files.md)
+   for information on how these files are formatted.
+
+## MacOS
+
+[Cyperduck](https://cyberduck.io) is a free, open source SFTP client for MacOS.
```
Comment (krivard) [inline docs/symptom-survey/server-access.md:46]:
```suggestion
[Cyberduck](https://cyberduck.io) is a free, open source SFTP client for MacOS.
```

File: docs/symptom-survey/server-access.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+(Download directly from their site for free; the App Store version costs $24.)
+
+1. Once the program is installed, click “Open Connection”.
+
+    ![Screenshot of Cyberduck's opening screen.](screenshots/cyberduck-1.png)
```
Comment (krivard) [inline docs/symptom-survey/server-access.md:51]: Is there a way to scale these down? They are perhaps excessively larger-than-life

**Review by krivard — APPROVED** [review]
One hilarious typo, minor style suggestions

File: docs/symptom-survey/server-access.md (hunk 3 above)
Comment (capnrefsmmat) [reply docs/symptom-survey/server-access.md:51]: They automatically have `max-width: 100%` in the CSS, and the text is kept to a reasonable column width, so I think this looks decent in the rendered version

**Pull request merged**